core/embed/rust/src/protobuf/defs.rs

```diff
@@ -127,7 +127,7 @@ pub fn find_name_by_msg_offset(msg_offset: u16) -> Result<u16, Error> {
         .filter(|def| def.msg_offset == msg_offset)
         .next()
         .map(|def| def.msg_name)
-        .ok_or(Error::Missing)
+        .ok_or(Error::KeyError(msg_offset.into()))
```

Comment:
`ok_or_else`

Reply:
fixed in b406bd0a0ff986392c2fccef336a7b9201734023